packages/ui-library/src/components/forms/textarea/index.ts

```diff
@@ -227,27 +225,20 @@ export class BlrTextarea extends LitElement {
             @blur=${this.handleBlur}
             @select=${this.handleSelect}
             @keyup=${this.updateCounter}
-          >
-${this.value}
-          </textarea
-          >
+          >${this.value}</textarea>
```

Comment:
I just found that the lit docs warn that you shouldn't put the text value as a child inside the textarea element in the first place: https://lit.dev/docs/templates/expressions/#invalid-locations

Instead it is suggested to pass it as `.value` attribute:

```suggestion
            .value=${this.value}
          ></textarea>
```

As a side effect, prettier shouldn't mess it up anymore too 🙂